user/build-environment-updates/2016-12-01.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2016-12-01/
+category: build_env_updates
+date: 2016-12-01
+---
+
+## 2016-12-01
+
+### Schedule
+
+[2016-12-01 15:00 UTC](http://everytimezone.com/#2016-12-1,180,cn3)
+
+### Platforms
+
+This update applies to fully-virtualized Ubuntu Trusty and Precise.
+
+### Added
+
+- MySQL 5.6 (Trusty only)
+- PostgreSQL 9.6 + postgis scripts packages (Trusty only)
+- PostgreSQL 9.5 + postgis scripts packages (Precise only)
+- APT settings to retain old configs on update (Trusty only)
+
+### Changed
+
+- Preinstalled Python versions to latest point releases, including CPython and
+  PyPy. (Trusty only)
+- Update to pyenv 1.0.0 (Trusty only)
+- Update to git 2.x (Trusty only)
+- Update to latest git-lfs (Trusty only)
+- Update to docker-compose 1.8.1, docker 1.12.3 (Trusty only)
+- Update to latest jdk\_switcher (Trusty only)
+
+### Removed
+
+- Preinstalled older versions of ruby from the Precise ruby stack.  All versions
+  remain installable on demand via `rvm`.
+- Preinstalled older versions of go on Precise ruby/default stack and all Trusty
+  stacks.  All versions supported by `gimme` may be installed on demand.
+- PostgreSQL 9.1 (Trusty only)
+- nexus.codehaus.org Maven mirror (Precise only)
+
+### Fixed
+
+- Include at least one JDK and the `jdk_switcher` tool in all Trusty stacks.
+- Preinstall ruby 2.2.5 for use with build caching and deployment tools.
+- Force removal of `~travis/.pearrc` at the end of image baking to ensure PHP is
+  less confused.
+
+### Details
+
+This update is a promotion of `group: edge` to `group: stable`.  We will retain
+the previous `group: stable` as `group: legacy` for at least a month following
+this change.  **NOTE**: When targeting Trusty, some `language:` declarations may
+result in fewer features than are listed here, as they may be routed to the
+"minimal" stack image.
```

Comment:
Think this could be more concrete: Which languages are affected?
And less concrete: Some way to avoid mentioning image stacks? Or should this be elaborated on someplace else?